library/src/main/java/pl/rspective/voucherify/android/client/api/VoucherifyApi.java

```diff
@@ -13,4 +14,7 @@
     @GET("/client/v1/validate")
     VoucherResponse validateVoucher(@QueryMap Map<String, String> queryParams);
 
+    @POST("/client/v1/redeem")
+    VoucherRedemptionResult redeemVoucher(@Query("code") String code, @Query("tracking_id") String trackingId);
```

Comment:
What about payload? This method should take RedemptionContext if you want to redeem gift vouchers or vouchers with validation rules.